src/utils/request-util.ts

```diff
@@ -24,6 +29,18 @@ export function createXmlRequestConfig(requestName: string, requestNamespace: st
   }
 }
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function generateRequestNode(paramOrder: RequestParamOrder, paramsBody: any) {
+  return `<${paramOrder.name}>${paramOrder.params
```

Comment:
My dream is that someday we will use xml2js instead of string manipulation. ☁️ 😄

Reply:
The problem is, that JSON doesn't guarantee the order, and the XML recipient does require it. So, I am not very hopeful about it...